src/ActionList/Item.tsx

```diff
@@ -125,6 +125,7 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
         ':hover:not([aria-disabled])': {
           backgroundColor: `actionListItem.${variant}.hoverBg`,
           color: getVariantStyles(variant, disabled).hoverColor,
+          boxShadow: `inset 0 0 0 max(1px, 0.0625rem) ${theme?.colors.border.muted}`,
```

Comment:
```suggestion
          boxShadow: `inset 0 0 0 max(1px, 0.0625rem) ${theme?. actionListItem.default. activeBorder}`,
```

I'm not sure if that syntax is right, maybe its:

```
boxShadow: `inset 0 0 0 max(1px, 0.0625rem) actionListItem.default. activeBorder`,
```

Regardless, this should use the ActionList specific color token which you can also see in the Primer View Components version 😄

Reply:
I think with the `sx` prop it needs to either be _just_ the token or uses that theme from context, potentially? So with the new token it would be:

```suggestion
          boxShadow: `inset 0 0 0 max(1px, 0.0625rem) ${theme?.colors.actionListItem.default.activeBorder}`,
```

Reply:
Me and @broccolinisoup went back and forth with the colors and got confused! Thanks for this input. I have updated the PR!